__tests__/endpoint.tests.js

```diff
@@ -103,6 +103,22 @@ describe('GET/api/articles', () => {
         expect(result.body.articles).toBeSortedBy('created_at', {descending:true});
         expect(Object.keys(result.body.articles[0])).toEqual(expectedColumns);
     })
+    it('200: accepts a topic query which filters articles by topic', async () => {
+        const result = await request(app).get('/api/articles?topic=cats');
+        const expectedColumns = ['article_id', 'title', 'topic', 'author', 'created_at', 'votes', 'article_img_url', 'comment_count'];
+
+        expect(result.status).toBe(200);
+        expect(result.body.articles).toHaveLength(1);
+        expect(result.body.articles).toBeSortedBy('created_at', {descending:true});
+        expect(Object.keys(result.body.articles[0])).toEqual(expectedColumns);
+    })
```

Comment:
We need to check that the articles coming back are about cats.

Reply:
We will also need a test for a topic that exists but returns an empty array as it has no articles.